### ISSUE TITLE
@Field(type = FieldType.Keyword) does not seem to be taken in account on Id fields

### ISSUE THREAD

**Issue by Tcharl** [VanRoy/spring-data-jest#96]
A solution for databases that are using uuid as Id (i.e. cassandra/mongo) and Spring-data-elastic is to use '@Field(type = FieldType.Keyword) ' in addition of the usual @Id field on the document.

It looks like it is not supported by Jest for the moment, the only solution being a json mapping:

```
@Id
    @Field(type = FieldType.Keyword)
    @Mapping(mappingPath = "/mappings/asset_id_mapping.json")// due to @Field(type = FieldType.Keyword) property ignored by Jestclient
    private String id;
{
    "type": "keyword",
    "fields": {
        "keyword": {
            "type": "keyword",
            "ignore_above": 256
        }
    }
}
```

**Comment by VanRoy** [comment]
Hi @Tcharl , Thanks for your feedback. I will investigate on it.

**Comment by VanRoy** [comment]
Hi @Tcharl , it's fixed in the new release 3.2.0.RELASE.

**Comment by Tcharl** [comment]
Whouhooooo! You're the one! Let's upgrade #jhipster-dependencies on my side :-D
